internal/weather/display.go

```diff
@@ -67,21 +85,27 @@ func DisplayWeather(weather *Weather, config Config) {
 		weatherID = weather.Weather[0].ID
 	}
 
+
 	// Determine units based on config
-	var windSpeedUnits, tempUnit string
+	var windSpeedUnits, tempUnit, dewPointUnit string
```

Comment:
Can't we just use the `tempUnit` as it has the same purpose and cannot be different?